src/service/url-replacements-impl.js

```diff
@@ -426,7 +430,7 @@ export class UrlReplacements {
    * @template T
    */
   getAccessValue_(getter, expr) {
-    return this.getAccessService_().then(accessService => {
+    return this.getAccessService_(this.win_).then(accessService => {
```

Comment:
Would it just be easier to bind in constructor? It will also ensure that we don't pass a wrong `@this` to that method.